cheroot/test/test_ssl.py

```diff
@@ -95,11 +97,34 @@ def respond(self):
         return super(HelloWorldGateway, self).respond()
 
 
-def make_tls_http_server(bind_addr, ssl_adapter, request):
+class WSGIEnvironmentGateway(Gateway_10):
```

Comment:
Why don't you just add this logic into `HelloWorldGateway` under a different URL (like `/env`).

Reply:
merged